virttest/utils_misc.py

```diff
@@ -3009,7 +3062,12 @@ def get_image_snapshot(image_file):
     1         1460096943             3.1M 2016-04-08 14:29:03   00:00:00.110
     """
     try:
+        avoid_qemu_lock_param = " -U"
         cmd = "qemu-img snapshot %s -l" % image_file
+        if compare_qemu_version(2, 10, 0):
+            # Currently the qemu lock only introduced in qemu-kvm-rhev,
```

Comment:
only introduced --> is introduced in qemu-kvm-rhev only

Reply:
fixed

Reply:
fixed